Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -19,24 +19,52 @@ struct GenerateCommand: ParsableCommand, HasTrackableParameters {
     )
     var path: String?
 
+    @Argument(help: """
+    A list of targets in which you'd like to focus. \
+    Those and their dependant targets will be generated as sources. \
+    If no target is specified, the project defined targets will be focused.
+    """)
+    var sources: [String] = []
+
+    @Flag(
+        name: .shortAndLong,
+        help: "Don't open the project after generating it."
+    )
+    var noOpen: Bool = false
+
     @Flag(
+        name: [.customShort("x"), .long],
+        help: "When passed it uses xcframeworks (simulator and device) from the cache instead of frameworks (only simulator)."
+    )
+    var xcframeworks: Bool = false
+
+    @Option(
         name: [.customShort("P"), .long],
-        help: "Only generate the local project (without generating its dependencies)."
+        help: "The name of the cache profile to be used when focusing on the target."
     )
-    var projectOnly: Bool = false
+    var profile: String?
```

Comment:
I think this makes sense but wonder if using the flag cache would be easier to understand

Reply:
Not sure I got your comment :) BTW it is just a porting of the arguments we had in `tuist focus`